readme.md

```diff
@@ -1,7 +1,7 @@
 # Antisnake
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_antisnake/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_antisnake/report/html_report/)
+- [DEMO LINK](https:/clonvhs.github.io/layout_antisnake/)
+- [TEST REPORT LINK](https://clonvhs.github.io/layout_antisnake/report/html_report/)
```

Comment:
add slash in demo link because the link doesn't work

Reply:
Thanks, fixed